user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleContentComponent.java

```diff
@@ -85,6 +102,56 @@ private void reloadOnBatchRegistration() {
         event -> displayComponentInContent(sampleDetailsComponent));
   }
 
+  private void addBatchUpdateListeners() {
+    batchDetailsComponent.addBatchCreationListener(this::registerBatch);
+    batchDetailsComponent.addBatchDeletionListener(this::openBatchDeletionConfirmation);
+    batchDetailsComponent.addBatchEditListener(this::editBatch);
+    batchDetailsComponent.addBatchViewListener(this::viewBatch);
+  }
+
+  private void registerBatch(CreateBatchEvent createBatchEvent) {
+    onRegisterButtonClicked();
```

Comment:
You can inline this method in a lambda. Naming the event `ignored` would indicate that the content of the event is ignored but the event triggers some action.

Reply:
done